rapid_response_xblock/models.py

```diff
@@ -41,3 +41,22 @@ def __str__(self):
                 answer_id=self.answer_id,
             )
         )
+
+
+class RapidResponseBlockStatus(models.Model):
```

Comment:
Should we use a `TimeStampedModel` here, or is that something we should tackle when we add the foreign key later?

Reply:
That's fine with me, but it seemed like there wasn't any reason to keep track of the timestamps for these records